pkg/pillar/cmd/volumemgr/handledownloader.go

```diff
@@ -58,12 +60,13 @@ func AddOrRefcountDownloaderConfig(ctx *volumemgrContext, blob types.BlobStatus)
 	// try to reserve storage, must be released on error
 	size := blob.Size
 	n := types.DownloaderConfig{
-		DatastoreIDList: blob.DatastoreIDList,
-		Name:            blob.RelativeURL,
-		ImageSha256:     blob.Sha256,
-		Size:            size,
-		Target:          locFilename,
-		RefCount:        refCount,
+		DatastoreIDList:        blob.DatastoreIDList,
+		Name:                   blob.RelativeURL,
+		ImageSha256:            blob.Sha256,
+		Size:                   size,
+		Target:                 locFilename,
+		RefCount:               refCount,
+		BlobDownloadRetryCount: blobDownloadRetryCount,
```

Comment:
FWIW I find the "blob" part of the field and variable names out of place. Nowhere in the downloader code do we refer to "blobs". That is a concept and name which belongs in the volumemgr. So please drop "blob" from these names.